### PR TITLE
Fix some typos and nits

### PR DESCRIPTION
**Pull request by standardAI**
This pull request fixes the argument in the np.clip() function call in the TabularMetrics class. It replaces "y_pred" with "temp_pred" to ensure correct clipping of values. Additionally, it removes trailing whitespace and fixes a typo in the rejected_text column.
@abhishekkrthakur

### CODE REVIEW

**Comment by standardAI** [comment]
@abhishekkrthakur

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -6,4 +6,4 @@ contact_links:
     about: Getting started and FAQs
   - name: AutoTrain Discussions
     url: https://github.com/huggingface/autotrain-advanced/discussions
-    about: General usage questions and community discussions
\ No newline at end of file
+    about: General usage questions and community discussions
```
Comment (abhishekkrthakur) [inline .github/ISSUE_TEMPLATE/config.yml:9]: remove this change

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Documentation is available [here](https://hf.co/docs/autotrain/).
 You can Install AutoTrain-Advanced python package via PIP. Please note you will need python >= 3.8 for AutoTrain Advanced to work properly.
 
     pip install autotrain-advanced
-    
+
```
Comment (abhishekkrthakur) [inline README.md:19]: remove this change

File: docs/source/dreambooth.mdx [hunk 3]
```diff
@@ -9,4 +9,4 @@ The data format for DreamBooth training is simple. All you need is images of a c
 To train a dreambooth model, please select an appropriate model from the hub.
 When choosing a model from the hub, please make sure you select the correct image size compatible with the model.
 
-Your concept token is `prompt` in parameters section.
\ No newline at end of file
+Your concept token is `prompt` in parameters section.
```
Comment (abhishekkrthakur) [inline docs/source/dreambooth.mdx:12]: remove this change. not an actual improvement

File: docs/source/index.mdx [hunk 4]
```diff
@@ -27,4 +27,4 @@ Using AutoTrain Advanced, advanced users can also control the hyperparameters us
 
 Everything else is the same as AutoTrain. You can use AutoTrain Advanced to train models for NLP, CV, Speech and Tabular tasks.
 
-We recommend using [AutoTrain Advanced](https://huggingface.co/new-space?template=autotrain-projects/autotrain-advanced) since it is faster, more flexible and will have more supported tasks and features in the future.
\ No newline at end of file
+We recommend using [AutoTrain Advanced](https://huggingface.co/new-space?template=autotrain-projects/autotrain-advanced) since it is faster, more flexible and will have more supported tasks and features in the future.
```
Comment (abhishekkrthakur) [inline docs/source/index.mdx:30]: no actual improvement. remove this

**Comment by standardAI** [comment]
All resolved.
@abhishekkrthakur

**Review by abhishekkrthakur — APPROVED** [review]

**Comment by abhishekkrthakur** [comment]
Thank you!

**Comment by standardAI** [comment]
Thanks for reviewing and merging!

**Pull request merged**